src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/SignAndAuthKeys.vue

```diff
@@ -38,11 +38,15 @@
         <xrd-search v-model="search" />
       </div>
     </div>
-    <div v-if="filtered && filtered.length < 1">
-      {{ $t('services.noMatches') }}
-    </div>
 
-    <template v-if="filtered">
+    <XrdEmptyPlaceholder
+      :data="filtered"
+      :loading="loading"
+      :filtered="search.length > 0"
```

Comment:
This causes error when `search` is undefined I.e. when you empty the search input with the `x` button

Reply:
Fixed